scripts/linters/js_ts_linter.py

```diff
@@ -1123,9 +1123,6 @@ def _get_trimmed_error_output(eslint_output):
             # and if that is True then we are replacing "error" with empty
             # string('') which is at the index 1 and message-id from the end.
             if re.search(r'^\d+:\d+', line.lstrip()):
-                # Replacing message-id with an empty string('').
-                if not 'Parsing error:' in line:
-                    line = re.sub(r'(\w+-*)+$', '', line)
```

Comment:
Same reason as above*

Reply:
Do you need to do a replace() call instead of this, like above?

Reply:
We are allowing printing the message-id in the error so we don't feed replacement for this re.sub(). Also, the .replace() call is happening on line 1130 & 1113-1116.

Reply:
This is all quite weird (that the code was there in the first place)... 

OK, sounds good -- thanks! No issues from my end.